packages/mdc-select/index.js

```diff
@@ -69,29 +71,40 @@ export class MDCSelect extends MDCComponent {
   }
 
   initialize(menuFactory = (el) => new MDCSimpleMenu(el)) {
-    this.menuEl_ = this.root_.querySelector('.mdc-select__menu');
+    this.surface_ = this.root_.querySelector(strings.SURFACE_SELECTOR);
+    this.label_ = this.root_.querySelector(strings.LABEL_SELECTOR);
+    this.bottomLine_ = this.root_.querySelector(strings.BOTTOM_LINE_SELECTOR);
+    this.selectedText_ = this.root_.querySelector(strings.SELECTED_TEXT_SELECTOR);
+    this.menuEl_ = this.root_.querySelector(strings.MENU_SELECTOR);
     this.menu_ = menuFactory(this.menuEl_);
-    this.selectedText_ = this.root_.querySelector('.mdc-select__selected-text');
+    this.ripple = new MDCRipple(this.surface_);
   }
 
   getDefaultFoundation() {
     return new MDCSelectFoundation({
       addClass: (className) => this.root_.classList.add(className),
       removeClass: (className) => this.root_.classList.remove(className),
+      addClassToLabel: (className) => this.label_.classList.add(className),
+      removeClassFromLabel: (className) => this.label_.classList.remove(className),
+      addClassToBottomLine: (className) => this.bottomLine_.classList.add(className),
+      removeClassFromBottomLine: (className) => this.bottomLine_.classList.remove(className),
+      setBottomLineAttr: (attr, value) => this.bottomLine_.setAttribute(attr, value),
       setAttr: (attr, value) => this.root_.setAttribute(attr, value),
       rmAttr: (attr, value) => this.root_.removeAttribute(attr, value),
-      computeBoundingRect: () => this.root_.getBoundingClientRect(),
+      computeBoundingRect: () => this.surface_.getBoundingClientRect(),
+      registerPointerDownHandler: (evtType, handler) => this.root_.addEventListener(evtType, handler),
+      deregisterPointerDownHandler: (evtType, handler) => this.root_.removeEventListener(evtType, handler),
       registerInteractionHandler: (type, handler) => this.root_.addEventListener(type, handler),
       deregisterInteractionHandler: (type, handler) => this.root_.removeEventListener(type, handler),
       focus: () => this.root_.focus(),
       makeTabbable: () => {
-        this.root_.tabIndex = 0;
+        this.menuEl_.tabIndex = 0;
       },
       makeUntabbable: () => {
-        this.root_.tabIndex = -1;
+        this.menuEl_.tabIndex = -1;
       },
-      getComputedStyleValue: (prop) => window.getComputedStyle(this.root_).getPropertyValue(prop),
-      setStyle: (propertyName, value) => this.root_.style.setProperty(propertyName, value),
+      getComputedStyleValue: (prop) => window.getComputedStyle(this.surface_).getPropertyValue(prop),
```

Comment:
That code is precisely the reason for changing this. The extra DOM node is needed so the visible surface of the `select` element is different from its container. We need this change because the new styles require `overflow: hidden` on the visible surface, which would be problematic for showing a menu. We need the new surface however for the new ripple and bottom line behaviors, which are what this method is concerned with